### PR TITLE
New package: UnknownSKL.Greenlight version 2.1.1

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137662)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by Trenly** [comment]
[Policy] Zip Binary

File: manifests/u/UnknownSKL/Greenlight/2.1.1/UnknownSKL.Greenlight.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: UnknownSKL.Greenlight
+PackageVersion: 2.1.1
```
Comment (GhoulBoii) [inline manifests/u/UnknownSKL/Greenlight/2.1.1/UnknownSKL.Greenlight.installer.yaml:5]:
```suggestion
PackageVersion: 2.1.1
InstallModes:
- interactive
- silent
- silentWithProgress
```

File: manifests/u/UnknownSKL/Greenlight/2.1.1/UnknownSKL.Greenlight.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: UnknownSKL.Greenlight
+PackageVersion: 2.1.1
+Installers:
+- InstallerUrl: https://github.com/unknownskl/greenlight/releases/download/v2.1.1/Greenlight-Setup-2.1.1.exe
+  Architecture: x86
```
Comment (GhoulBoii) [inline manifests/u/UnknownSKL/Greenlight/2.1.1/UnknownSKL.Greenlight.installer.yaml:8]:
```suggestion
  Architecture: x64
```
Shouldn't this be x64 architecture?

File: manifests/u/UnknownSKL/Greenlight/2.1.1/UnknownSKL.Greenlight.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: UnknownSKL.Greenlight
+PackageVersion: 2.1.1
+PackageLocale: en-US
+Publisher: UnknownSKL
+PackageName: Greenlight
+License: MIT
+Copyright: Copyright © 2021-2022 UnknownSKL
```
Comment (GhoulBoii) [inline manifests/u/UnknownSKL/Greenlight/2.1.1/UnknownSKL.Greenlight.locale.en-US.yaml:10]:
```suggestion
Copyright: Copyright © 2021-2024 UnknownSKL
```

**Review by ksharizard — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**